src/style.css

```diff
@@ -0,0 +1,141 @@
+/* #region Normalize */
+
+.page {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+
+  --link-height: 40px;
+  --color-dark: #060b35;
+}
+
+.page__content {
+  margin: 0;
+}
+
+.title {
+  font-size: 12px;
+  margin: 0;
+}
+
+.phrase {
+  margin: 0;
+  color: #616070;
+}
+
+/* #endregion */
+
+/* #region card */
+
+.card {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 10th item. Make separate files for each BEM-block